zip-0226.rst

```diff
@@ -325,6 +325,111 @@ A ZSA Asset Burn description is encoded in a transaction as an of a AssetBurn ty
 | 8     | valueBurn | :math:`\{1 .. 2^{64} - 1\}` | The amount of the AssetType being burnt.                                                                                  |
 +-------+-----------+-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
 
+TxId Digest
+===========
+
+The transaction digest algorithm defined in ZIP 244 [#zip-0244]_ is modified by the ZSA protocol to add a new branch for issuance information, and add an additional hash for the Asset Base within the ``orchard_digest``. The details of these changes are described in this section. The overall tree structure of the hash is as follows, and the additional names not referenced in ZIP 244 are described in detail below::
+
```

Comment:
Changes and additions to the Orchard protocol are styled in **Bold**.

Reply:
As discussed, now using `[UPDATED FOR ZSA]` to highlight this.

Reply:
Removed the big tree.